vector/src/main/java/im/vector/app/features/settings/devices/v2/rename/RenameSessionFragment.kt

```diff
@@ -62,12 +62,20 @@ class RenameSessionFragment :
     }
 
     private fun initEditText() {
-        views.renameSessionEditText.showKeyboard(andRequestFocus = true)
+        showKeyboard()
         views.renameSessionEditText.doOnTextChanged { text, _, _, _ ->
             viewModel.handle(RenameSessionAction.EditLocally(text.toString()))
         }
     }
 
+    private fun showKeyboard() {
```

Comment:
```suggestion
    private fun showKeyboard() {
        val focusChangeListener = object : ViewTreeObserver.OnWindowFocusChangeListener {
            override fun onWindowFocusChanged(hasFocus: Boolean) {
                if (hasFocus) {
                    views.renameSessionEditText.showKeyboard(andRequestFocus = true)
                }
                views.renameSessionEditText.viewTreeObserver.removeOnWindowFocusChangeListener(this)
            }
        }
        views.renameSessionEditText.viewTreeObserver.addOnWindowFocusChangeListener(focusChangeListener)
    }
```

Reply:
I think we should remove the listener once done (see my suggestion above). Otherwise it will keep adding a listener each time and every listeners will be called.

Reply:
Right, done.